<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Monitoring</string>
    </property>
   </widget>
   <widget class="QLabel" name="currencyValueLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>741</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Currency Value USD/RUB: N/A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openLinkButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open USD/RUB</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-30</y>
      <width>991</width>
      <height>721</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="eurusdValueLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Currency Value EUR/USD: N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="jpyusdValueLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Currency Value  JPY/USD: N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="zarusdValueLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Currency Value ZAR/USD: N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="usdrubIndicator">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="eurusdIndicator">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jpyusdIndicator">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zarusdIndicator">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="newLinkButton1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open  EUR/USD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newLinkButton2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open JPY/USD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newLinkButton3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open ZAR/USD</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>startButton</zorder>
   <zorder>currencyValueLabel</zorder>
   <zorder>openLinkButton</zorder>
   <zorder>eurusdValueLabel</zorder>
   <zorder>jpyusdValueLabel</zorder>
   <zorder>zarusdValueLabel</zorder>
   <zorder>usdrubIndicator</zorder>
   <zorder>eurusdIndicator</zorder>
   <zorder>jpyusdIndicator</zorder>
   <zorder>zarusdIndicator</zorder>
   <zorder>newLinkButton1</zorder>
   <zorder>newLinkButton2</zorder>
   <zorder>newLinkButton3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
